sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -1047,6 +1070,56 @@ class TransformWithStateSuite extends StateStoreMetricsTest
     }
   }
 
+  test("state data source integration - value state with single variable") {
```

Comment:
Shall we also have another test cases for value state with TTL?

Reply:
Yup - waiting for the vcf integration PR to merge